### DISCUSSION TITLE
some classes not working in nextjs+ tailwind project

### DISCUSSION THREAD

**Discussion by rutikwankhade** [tailwindlabs/tailwindcss#7692] [Help]
I am using nextjs tailwind starter from here
https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss

But some of the classes are not working in my app. for ex `text-4xl` would work but `text-5xl` doesn't, same with some padding and width classes. same classes are working in other some files but not in few. How do I fix this.

here is tailwind.config file

```
module.exports = {
  content: [
    './src/pages/**/*.{js,ts,jsx,tsx}',
    '.src/components/**/*.{js,ts,jsx,tsx}',
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}

```

nextjs latest version
 "tailwindcss": "^3.0.7",

**Comment by NovusOpus** [comment]
In the repo you shared, there's no `src` directory/path.

So unless you changed the name of the `pages` directory to `src` you need to update your `tailwind.config.js` to include the path where you're using Tailwind CSS Classes or Utilities, In this case assuming you're using the `pages` directory and also added a `components` directory/path for your react components this should work:

```js
module.exports = {
  content: [
    './pages/**/*.{js,ts,jsx,tsx}',
    './components/**/*.{js,ts,jsx,tsx}',
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}


```

**Reply by uancamilo** [reply]
so, thank you... yes, I was facing the same problem but in a little different way. this worked for me.
